packages/react-native-client/ios/VideoPreviewView.swift

```diff
@@ -13,11 +17,9 @@ class VideoPreviewView: ExpoView {
         addSubview(videoView!)
     }
 
-    override func willMove(toSuperview newSuperview: UIView?) {
-        super.willMove(toSuperview: newSuperview)
-        if newSuperview == nil {
-            localVideoTrack?.stop()
-        } else {
+    private func trySetLocalCameraTrack() {
+        DispatchQueue.main.async { [weak self] in
+            guard let self = self else { return }
```

Comment:
```suggestion
            guard let self else { return }
```
Since Swift 5.7 we can omit the other `self`